### DISCUSSION TITLE
Auto-regression model doesn't work ?

### DISCUSSION THREAD

**Discussion by theoeiferman** [ourownstory/neural_prophet#425] [Q&A - get help using NeuralProphet] [answered]
Hey, couldn't find anything online for this error : **Raise ValueError("cannot reindex from a duplicate axis")**

 When I am tring to fit the model to my simple dataset data_prophet  this way:

`model_AR.fit(data_prophet,valid_p=0.2,freq='H',epochs=100)`
 
where

`model_AR =NeuralProphet(n_forecasts=60,n_lags=60,changepoints_range=0.95,n_changepoints=100,yearly_seasonality=True,weekly_seasonality=False,daily_seasonality=False,batch_size=64,epochs=100,learning_rate=1.0)`

What's wrong here ?
Thans you in advance for your help

**Comment by ourownstory** [comment, marked as answer]
Hi @theoeiferman 
This sounds like a data format issue.  Might you have an index in your dataframe? 
You can try creating a fresh dataframe without an index and just 'ds' and 'y' columns.
If that does not resolve it, please share some details about your dataframe.
